### PR TITLE
LPS-188648 [MARKETPLACE] Implement Replace For DDM Template

### PR DESCRIPTION
**Pull request by joaodamasceno2001**

### CODE REVIEW

**Comment by joaodamasceno2001** [comment]
ci:test:sf

**Comment by nNilton** [comment]
ci:forward:force

**Comment by silasbsi** [comment]
Just starting reviewing :)

**Comment by joaodamasceno2001** [comment]
ci:test:sf

File: modules/apps/site-initializer/site-initializer-extender/site-initializer-extender/src/main/java/com/liferay/site/initializer/extender/internal/BundleSiteInitializer.java [hunk 1]
```diff
@@ -3834,7 +3884,8 @@ else if (StringUtil.equals(groupType, "UserGroups")) {
 	private void _addSegmentsExperiences(
 			Map<String, String> assetListEntryIdsStringUtilReplaceValues,
 			Map<String, String> clientExtensionEntryIdsStringUtilReplaceValues,
-			Map<String, String> ddmStructureEntryIdsStringUtilReplaceValues,
+			Map<String, String> ddmTemplateIdsStringUtilReplaceValues,
+			Map<String, String> ddmStructureIdsStringUtilReplaceValues,
```
Comment (nNilton) [inline modules/apps/site-initializer/site-initializer-extender/site-initializer-extender/src/main/java/com/liferay/site/initializer/extender/internal/BundleSiteInitializer.java:3888]: sort

File: modules/apps/site-initializer/site-initializer-extender/site-initializer-extender/src/main/java/com/liferay/site/initializer/extender/internal/BundleSiteInitializer.java [hunk 2]
```diff
@@ -3918,7 +3969,8 @@ private void _addSegmentsExperiences(
 					_addLayoutContent(
 						assetListEntryIdsStringUtilReplaceValues,
 						clientExtensionEntryIdsStringUtilReplaceValues,
-						ddmStructureEntryIdsStringUtilReplaceValues,
+						ddmTemplateIdsStringUtilReplaceValues,
+						ddmStructureIdsStringUtilReplaceValues,
```
Comment (nNilton) [inline modules/apps/site-initializer/site-initializer-extender/site-initializer-extender/src/main/java/com/liferay/site/initializer/extender/internal/BundleSiteInitializer.java:3973]: sort

File: modules/apps/site-initializer/site-initializer-extender/site-initializer-extender/src/main/java/com/liferay/site/initializer/extender/internal/BundleSiteInitializer.java [hunk 3]
```diff
@@ -1179,7 +1189,8 @@ private void _addLayoutsContent(
 			_addLayoutContent(
 				assetListEntryIdsStringUtilReplaceValues,
 				clientExtensionEntryIdsStringUtilReplaceValues,
-				ddmStructureEntryIdsStringUtilReplaceValues,
+				ddmTemplateIdsStringUtilReplaceValues,
+				ddmStructureIdsStringUtilReplaceValues,
```
Comment (nNilton) [inline modules/apps/site-initializer/site-initializer-extender/site-initializer-extender/src/main/java/com/liferay/site/initializer/extender/internal/BundleSiteInitializer.java:1193]: sort

File: modules/apps/site-initializer/site-initializer-extender/site-initializer-extender/src/main/java/com/liferay/site/initializer/extender/internal/BundleSiteInitializer.java [hunk 4]
```diff
@@ -1164,7 +1173,8 @@ private void _addLayoutPageTemplates(
 	private void _addLayoutsContent(
 			Map<String, String> assetListEntryIdsStringUtilReplaceValues,
 			Map<String, String> clientExtensionEntryIdsStringUtilReplaceValues,
-			Map<String, String> ddmStructureEntryIdsStringUtilReplaceValues,
+			Map<String, String> ddmTemplateIdsStringUtilReplaceValues,
+			Map<String, String> ddmStructureIdsStringUtilReplaceValues,
```
Comment (nNilton) [inline modules/apps/site-initializer/site-initializer-extender/site-initializer-extender/src/main/java/com/liferay/site/initializer/extender/internal/BundleSiteInitializer.java:1177]: sort

File: modules/apps/site-initializer/site-initializer-extender/site-initializer-extender/src/main/java/com/liferay/site/initializer/extender/internal/BundleSiteInitializer.java [hunk 5]
```diff
@@ -607,7 +611,8 @@ public void initialize(long groupId) throws InitializationException {
 				() -> _addSegmentsExperiences(
 					assetListEntryIdsStringUtilReplaceValues,
 					clientExtensionEntryIdsStringUtilReplaceValues,
-					ddmStructureEntryIdsStringUtilReplaceValues,
+					ddmTemplateIdsStringUtilReplaceValues,
+					ddmStructureIdsStringUtilReplaceValues,
```
Comment (nNilton) [inline modules/apps/site-initializer/site-initializer-extender/site-initializer-extender/src/main/java/com/liferay/site/initializer/extender/internal/BundleSiteInitializer.java:615]: sort

File: modules/apps/site-initializer/site-initializer-extender/site-initializer-extender/src/main/java/com/liferay/site/initializer/extender/internal/BundleSiteInitializer.java [hunk 6]
```diff
@@ -594,7 +597,8 @@ public void initialize(long groupId) throws InitializationException {
 				() -> _addLayoutsContent(
 					assetListEntryIdsStringUtilReplaceValues,
 					clientExtensionEntryIdsStringUtilReplaceValues,
-					ddmStructureEntryIdsStringUtilReplaceValues,
+					ddmTemplateIdsStringUtilReplaceValues,
+					ddmStructureIdsStringUtilReplaceValues,
```
Comment (nNilton) [inline modules/apps/site-initializer/site-initializer-extender/site-initializer-extender/src/main/java/com/liferay/site/initializer/extender/internal/BundleSiteInitializer.java:601]: sort

File: modules/apps/site-initializer/site-initializer-extender/site-initializer-extender/src/main/java/com/liferay/site/initializer/extender/internal/BundleSiteInitializer.java [hunk 7]
```diff
@@ -1749,16 +1760,41 @@ private Map<String, String> _addOrUpdateDDMStructures(
 		return ddmStructuresIdsStringUtilReplaceValues;
 	}
 
-	private void _addOrUpdateDDMTemplates(
+	private Map<String, String> _addOrUpdateDDMTemplates(
 			DDMStructureLocalService ddmStructureLocalService,
 			ServiceContext serviceContext)
 		throws Exception {
 
+		Map<String, String> ddmTemplateIdsStringUtilReplaceValues =
```
Comment (nNilton) [inline modules/apps/site-initializer/site-initializer-extender/site-initializer-extender/src/main/java/com/liferay/site/initializer/extender/internal/BundleSiteInitializer.java:1768]: this map does not contains just ddm templates anymore
please rename to following the existing pattern like: ddmTemplateIdsAndTemplateEntryIdsStringUtilReplaceValues

**Review by nNilton — CHANGES_REQUESTED** [review]

**Comment by joaodamasceno2001** [comment]
ci:test:sf

**Comment by joaodamasceno2001** [comment]
ci:test:sf

**Comment by joaodamasceno2001** [comment]
ci:test:sf

**Comment by nNilton** [comment]
ci:test:relevant

**Comment by joaodamasceno2001** [comment]
ci:reevaluate:1267401_10216

**Comment by nNilton** [comment]
ci:reevaluate:1267401_10216

**Comment by nNilton** [comment]
ci:test:relevant

**Comment by nNilton** [comment]
ci:reevaluate:1317401_8586

**Comment by nNilton** [comment]
ci:forward:force

**Pull request closed without merging**